ruff_lsp/server.py

```diff
@@ -821,26 +829,44 @@ async def code_action(params: CodeActionParams) -> list[CodeAction] | None:
     first, and if there's no cell with the given URI, it will fallback to the text
     document.
     """
+
+    def document(uri: str, kind: str) -> Document:
```

Comment:
Can we call this something else? e.g.`document_from` or `load_document` or `document_from_kind` or `get_document` 

I mistook it for a variable instead of a function later on.